src/bodypart.cpp

```diff
@@ -227,26 +227,25 @@ void body_part_struct::check() const
     }
 }
 
+// Some languages do not have plural forms, so we don't use ngettext()/npgettext()
+// in the 3 following methods so the singular and plural strings can be properly
+// translated for these languages.
 std::string body_part_name( body_part bp, int number )
 {
     const auto &bdy = get_bp( bp );
-    return ngettext( bdy.name.c_str(),
-                     bdy.name_multiple.c_str(), number );
+    return _( number > 1 ? bdy.name_multiple : bdy.name );
```

Comment:
This looks odd. Wouldn't it be better to write like so?
```cpp
    return number > 1 ? _( bdy.name_multiple ) : _( bdy.name );
```

Reply:
Done